app/repos/cdo.js

```diff
@@ -268,6 +278,14 @@ const getSummaryCdos = async (filter) => {
     }
   }
 
+  const order = []
+
+  if (sortKey !== undefined) {
+    order.push([sequelize.col(sortKey), sortOrder])
+  }
+
+  order.push([sequelize.col('registration.cdo_expiry'), sortOrder])
```

Comment:
So if you supply a sort order (ASC/DESC), the direction will be applied to BOTH the supplied column (if you supply a column) and to the cdo_expiry? Is it not an either/or i.e. if supplied, sort by supplied column. If not supplied, sort by cdo_expiry?